.chloggen/gitprovider-impl-2.yaml

```
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: new_component
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: gitproviderreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: add repo, branch, and contributor count metrics as second implementation pull request
```

No need to mention this explicitly in the changelog IMO

```suggestion
note: add repo, branch, and contributor count metrics
```